tests/p2p/test_peer_id.py

```diff
@@ -149,14 +149,20 @@ def test_retry_connection(self):
         self.assertEqual(p.retry_interval, 5)
         self.assertEqual(p.retry_timestamp, 0)
 
+
+class BasePeerIdTest(unittest.TestCase):
+    __test__ = False
+
     @inlineCallbacks
     def test_validate_entrypoint(self):
         manager = self.create_peer('testnet', unlock_wallet=False)
         peer_id = manager.my_peer
         peer_id.entrypoints = ['tcp://127.0.0.1:40403']
 
         # we consider that we are starting the connection to the peer
-        protocol = HathorProtocol('testnet', peer_id, None, node=manager, use_ssl=True, inbound=False)
+        # TODO: also test with sync-v2 enabled
+        protocol = HathorProtocol('testnet', peer_id, None, node=manager, use_ssl=True, inbound=False,
+                                  enable_sync_v1=True, enable_sync_v2=False)
```

Comment:
Should we use `self._enable_sync_v1` and `self._enable_sync_v2`?

Reply:
Yes. I missed this. Fixed.